acceptance/acceptance_test.go

```diff
@@ -2929,6 +2929,56 @@ include = [ "*.jar", "media/mountain.jpg", "/media/person.png", ]
 				})
 			})
 		})
+
+		when("builder create", func() {
+			when("--flatten=<buildpacks>", func() {
+				it("should flatten together all specified buildpacks", func() {
```

Comment:
The test **correctly failed** when using previous pack version to create the builder. That's the reason the error was reported.